### PR TITLE
Server: Remove node user

### PR DESCRIPTION
**Pull request by ifurther**
Maybe could remove the node user in finally stage.

### CODE REVIEW

**Comment by laurent22** [comment]
Or just use this user instead of the "joplin" user

**Comment by tessus** [comment]
I thought there was an issue by removing the joplin user, because bind mounts would run under that user. An upgrade could break people's setups.

**Comment by ifurther** [comment]
Or just using this request to add joplin user in node group, it could mount volume in compose mode

File: Dockerfile.server [hunk 1]
```diff
@@ -56,12 +56,12 @@ RUN BUILD_SEQUENCIAL=1 yarn install --inline-builds \
...
 ARG user=joplin
-RUN useradd --create-home --shell /bin/bash $user
+RUN usermod -d /home/joplin -l joplin node
 
 USER $user
 
-COPY --chown=$user:$user --from=builder /build/packages /home/$user/packages
-COPY --chown=$user:$user --from=builder /tini /usr/local/bin/tini
+COPY --chown=$user:node --from=builder /build/packages /home/$user/packages
+COPY --chown=$user:node --from=builder /tini /usr/local/bin/tini
```
Comment (tessus) [inline Dockerfile.server:64]: I don't understand what is going on here. First you rename the `node` user to `joplin` (hardcoded) and then you change the group owner to `node`.

**Review by tessus — COMMENTED** [review]
Please see my comment inline.

File: Dockerfile.server (hunk 1 above)
Comment (ifurther) [reply Dockerfile.server:64]: If using the user setting the volume's own is node

Comment (tessus) [reply Dockerfile.server:64]: I think we are talking past each other. I know what chown does. 

Look at line 59: `RUN usermod -d /home/joplin -l joplin node` is the problem.

**Comment by laurent22** [comment]
If there are issues with bind mounts maybe we should go back to the initial commit, and just remove the node user?

**Comment by laurent22** [comment]
No idea what you're doing, now you moved lines around with no explanations.

**Comment by tessus** [comment]
I think we can close this PR. Somehow the current code looks the same as before after 9 commits. 

Next time you open a PR, please use a detailed description of your changes and what they are solving.

**Pull request closed without merging**